samples/dect/dect_phy/dect_shell/src/dect/common/dect_phy_api_scheduler.c

```diff
@@ -279,6 +279,23 @@ dect_phy_api_scheduler_list_item_remove_by_phy_op_handle(uint16_t handle)
 	return iterator;
 }
 
+bool dect_phy_api_scheduler_list_item_running_by_phy_op_handle(uint16_t handle)
```

Comment:
I don't see this function being used anywhere.

Reply:
Usage is coming.